src/phoenix/server/api/routers/v1/datasets.py

```diff
@@ -35,10 +35,39 @@
 
 from phoenix.db import models
 from phoenix.db.insertion.dataset import DatasetAction, add_dataset_examples
+from phoenix.schemas.core import Dataset
 
 logger = logging.getLogger(__name__)
 
 
+async def get_dataset_by_id(request: Request) -> Response:
+    """
+    summary: Get dataset by ID
+    operationId: getDatasetById
+    tags:
+      - datasets
+    parameters:
+      - in: path
+        name: id
+        required: true
+        schema:
+          type: integer
+    responses:
+      200:
+        description: Success
+      404:
+        description: Dataset not found
+    """
+    dataset_id = int(request.path_params["id"])
+    async with request.app.state.db() as session:
+        dataset = await session.get(models.Dataset, dataset_id)
+        if dataset is None:
```

Comment:
```suggestion
        if (dataset := await session.get(models.Dataset, dataset_id)) is None:
```

Reply:
I felt the one liner was harder to read given its length when writing it out